router.go

```diff
@@ -45,10 +45,10 @@ func AddActionViaAST(actions []Action, routerFilePath string, destDir string) {
 	// delete return statement
 	fn.Body.List = fn.Body.List[:numStatements-1]
 
-	controllerStmt := NewControllerStatement(actions[0].Resource)
+	controllerStmt := NewControllerStatement(data.Name)
```

Comment:
if there are no actions there's probably something wrong with how the user is designing their oa/swagger spec or there's a bug in the non oa/swagger code. either way, this reads less magically.